CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 ### Added
```

Comment:
Maybe lets just put a date at the top and a fresh template, especially since this basically the MVP, and then we can keep track of changes over time.